Default (Linux).sublime-keymap

```diff
@@ -5,5 +5,6 @@
     "context": [
       { "key": "selector", "operator": "equal", "operand": "text.csearch" }
     ]
-  }
+  },
+  { "keys": ["alt+ctrl+shift+f"], "command": "csearch" }
```

Comment:
I prefer not adding this keyboard shortcut to the plugin. I've run into too many conflicts in the past, be it with another plugin or a user having already defined one. Instead, I like to let the user define it if necessary.

But I'm willing to be convinced otherwise if you have strong feelings on it and raise some good points.